harness/determined/cli/master.py

```diff
@@ -20,6 +20,10 @@ def show_config(args: Namespace) -> None:
 
 @authentication.required
 def set_master_config(args: Namespace) -> None:
+    cli.warn(
+        "WARNING: This will only make ephermeral changes to the master config, "
```

Comment:
this probably doesn't need the `WARNING:` prefix given it is `cli.warn`, but mlsys will need to review this anyway.

Reply:
`cli.warn` prints the message in yellow without the prefix. throughout the CLI we usually just print warnings in yellow without any prefix. 

this particular warning does not strike me as a _very bad_ warning, so I'd be in favor of removing the prefix.